src/main/flight/mixer.c

```diff
@@ -765,6 +765,10 @@ void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensation)
     }
 
     motorMixRange = motorMixMax - motorMixMin;
+    if( targetPidLooptime && throttle_boost > 0.0f) {
```

Comment:
is `targetPidLooptime` necessary here? filter is already initialized, so it's probably too late anyway

Reply:
That's true. In my prototype I used to initialize the filter each time here, that's why it's still there. I'll take it out.